modules/tinymce/src/core/main/ts/api/commands/ClipboardCommands.ts

```diff
@@ -1,38 +1,102 @@
+import { PlatformDetection } from '@ephox/sand';
+
 import Editor from '../Editor';
 import Env from '../Env';
 
 export const registerCommands = (editor: Editor): void => {
-  editor.editorCommands.addCommands({
-    'Cut,Copy,Paste': (command) => {
-      const doc = editor.getDoc();
-      let failed;
+  const browser = PlatformDetection.detect().browser;
```

Comment:
`PlatformDetection.detect()` is cached, you can just call it every time rather than cache the browser value. Or just use `Env.browser`, since that's already imported.